### PR TITLE
Imaging Preprocessing: Order the acquisitions per modality based on series number: Redmine 10959 

### PR DESCRIPTION
**Pull request by MounaSafiHarab**
Related to: https://github.com/aces/Loris-MRI/pull/160

### CODE REVIEW

File: SQL/0000-00-00-schema.sql [hunk 1]
```diff
@@ -495,6 +495,7 @@ CREATE TABLE `files` (
   `Caveat` tinyint(1) default NULL,
   `TarchiveSource` int(11) default NULL,
   `ScannerID` int(10) unsigned NOT NULL default '0',
+  `AcqOrderPerModality` int(11) default NULL,
```
Comment (MounaSafiHarab) [inline SQL/0000-00-00-schema.sql]: @driusan 
I randomly picked int(11); what is reasonable you think?

**Comment by driusan** [comment]
@MounaSafiHarab This can't be merged because of a conflict.

**Comment by MounaSafiHarab** [comment]
@driusan 
I did rebase. But I am still waiting for @cmadjar to test with the associated Loris-MRI pull request...

File: SQL/0000-00-00-schema.sql [hunk 2]
```diff
@@ -495,6 +495,7 @@ CREATE TABLE `files` (
   `Caveat` tinyint(1) default NULL,
   `TarchiveSource` int(11) default NULL,
   `ScannerID` int(10) unsigned default NULL,
+  `AcqOrderPerModality` int(11) default NULL,
```
Comment (cmadjar) [inline SQL/0000-00-00-schema.sql]: That field should probably be called: 'ModalityOrder' unless I misunderstood to match the code in LORIS-MRI (unless I misunderstood the code)

File: SQL/Archive/17.0/2016-09-20-OrderOfAcquisition [hunk 3]
```diff
@@ -0,0 +1 @@
+ALTER TABLE files ADD `AcqOrderPerModality` int(11) default NULL;
```
Comment (cmadjar) [inline SQL/Archive/17.0/2016-09-20-OrderOfAcquisition]: Same comment here

File: SQL/0000-00-00-schema.sql (hunk 2 above)
Comment (MounaSafiHarab) [inline SQL/0000-00-00-schema.sql]: so @driusan 
changes were done on the LORIS-MRi side to refer to this column name here.

**Pull request merged**